src/rqt_publisher/publisher_widget.py

```diff
@@ -33,19 +33,20 @@
 from __future__ import division
 import os
 
+from ament_index_python import get_resource, get_resources, has_resource
+
 from python_qt_binding import loadUi
 from python_qt_binding.QtCore import Signal, Slot
 from python_qt_binding.QtGui import QIcon
 from python_qt_binding.QtWidgets import QWidget
 
-import roslib
-import rosmsg
-import rospkg
-import rospy
-
+from qt_gui.ros_package_helper import get_package_path
 from qt_gui_py_common.worker_thread import WorkerThread
-from rqt_py_common.extended_combo_box import ExtendedComboBox
+
+import rclpy
 from .publisher_tree_widget import PublisherTreeWidget
+from rqt_py_common.extended_combo_box import ExtendedComboBox
+from rqt_py_common.topic_helpers import get_message_class
```

Comment:
get_message_class is getting moved to `rqt_py_common.message_helpers`

Reply:
Fixed